util/src/main/java/com/liferay/faces/util/application/internal/ResourceDependencyHandlerImpl.java

```diff
@@ -13,21 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.liferay.faces.util.application;
+package com.liferay.faces.util.application.internal;
 
-/**
- * This interface defines a contract for getting information about a {@link javax.faces.component.UIComponent}} resource
- * such as its name, library, and whether or not it should be rendered.
- *
- * @author  Neil Griffin
- */
-public interface ComponentResource {
```

Comment:
Github is too smart for its own good. This is an API change: removed `ComponentResource`. It could be deprecated instead, but it is very low risk because non-Faces projects are probably not using it.